_includes/concepts.md

```diff
@@ -52,10 +52,10 @@ A client is anything connecting to a server that is not another server. Each cli
 
 Nicknames are non-empty strings with the following restrictions:
 
-- They MUST NOT contain any of the following characters: space `(' ', 0x20)`, comma `(',', 0x2C)`, asterisk `('*', 0x2A)`, question mark `('?', 0x3F)`, exclamation mark `('!', 0x21)`, at sign `('@', 0x40)`.
-- They MUST NOT start with any of the following characters: dollar `('$', 0x24)`, colon `(':', 0x3A)`.
+- They MUST NOT contain any of the following characters: space `(' ', 0x20)`, exclamation mark `('!', 0x21)`, quotation mark `('"', 0x22)`, hash `('#', 0x23)`, dollar sign `('$', 0x24)`, percent sign `('%', 0x25)`, ampersand `('&', 0x26)`, apostrophe `(''', 0x27)`, opening bracket `('(', 0x28)`, closing bracket `(')', 0x29)`, asterisk `('*', 0x2A)`, plus sign `('+', 0x2B)`, comma `(',', 0x2C)`, dot `('.', 0x2E)`, forward slash `('/', 0x2F)`, colon `(':', 0x3A)`, semicolon `(';', 0x3B)`, less-than sign `('<', 0x3C)`, equals sign `('=', 0x3D)`, greater-than sign `('>', 0x3E)`, question mark `('?', 0x3F)`, at sign `('@', 0x40)`.
+- They MUST NOT start with any of the following characters: hyphen-minus `('-', 0x2D)`, a digit `('0' - '9', 0x30 - 0x39)`.
 - They MUST NOT start with a character listed as a [channel type](#channel-types) prefix.
-- They SHOULD NOT contain any dot character `('.', 0x2E)`.
+- They MUST NOT start with a character listed as a [channel membership prefix](#channel-membership-prefixes), or prefix listed in the IRCv3 [`multi-prefix` Extension](https://ircv3.net/specs/extensions/multi-prefix).
 
 Servers MAY have additional implementation-specific nickname restrictions.
```

Comment:
In response to feedback, perhaps this suffices:

```suggestion
Servers MAY have additional and/or alternative implementation-specific nickname restrictions, e.g. allowing certain characters prohibited by the RFC to accommodate bridges.
```

Reply:
This effectively makes all of the MUST NOT above moot, and allows servers to completely break the protocol.